tests/acceptance/standard/test_branches.py

```diff
@@ -60,6 +60,36 @@ def test__protect_and_unprotect(self, project, branch):
         assert merge_access_user_ids is None
         assert unprotect_access_level is None
 
+    def test__protect_with_wildcard(self, project):
+        branch_wildcard = "r-*"
+        config_protect_branch = f"""
+        projects_and_groups:
+          {project.path_with_namespace}:
+            branches:
+              {branch_wildcard}:
+                protected: true
+                push_access_level: {AccessLevel.NO_ACCESS.value}
+                merge_access_level: {AccessLevel.MAINTAINER.value}
+                unprotect_access_level: {AccessLevel.MAINTAINER.value}
+        """
+
+        run_gitlabform(config_protect_branch, project.path_with_namespace)
+
+        (
+            push_access_levels,
+            merge_access_levels,
+            push_access_user_ids,
+            merge_access_user_ids,
+            _,
+            _,
+            unprotect_access_level,
+        ) = get_only_branch_access_levels(project, branch_wildcard)
+        assert push_access_levels == [AccessLevel.NO_ACCESS.value]
+        assert merge_access_levels == [AccessLevel.MAINTAINER.value]
+        assert push_access_user_ids == []
+        assert merge_access_user_ids == []
+        assert unprotect_access_level is AccessLevel.MAINTAINER.value
```

Comment:
What do you think of this. After `run_gitlabform`, create a branch that matches the wildcard. Then get the access config of that branch to see if it's been protected or not. Or, do you think that shouldn't be necessary?

Reply:
Yes I think thats probably worth doing, I'll try and give that a go

Reply:
This has now been added

Reply:
Thanks. Let's close this thread.